model/task/task.go

```diff
@@ -3320,6 +3320,11 @@ func (t *Task) Blocked() bool {
 	return false
 }
 
+// isScheduled returns if a task is not scheduled to run
```

Comment:
This function is called IsScheduled but seems to return if it's not scheduled (comment and code-wise); can we have this return if it is scheduled?